ProjectName/src/features/accountCard/StarAccount.style.ts

```diff
@@ -56,14 +36,34 @@ const AccountCardStyle = StyleSheet.create({
     justifyContent: 'space-between',
   },
   moneyText: {
-    fontSize: 20,
+    color: 'black',
 
+    fontFamily: 'Aspira-Medium',
+    fontSize: 20,
     fontWeight: '500',
     lineHeight: 24,
-    fontFamily: 'Aspira-Medium',
-    color: 'black',
     marginBottom: 14,
   },
+
+  starContainer: {
+    flexDirection: 'column',
+    flex: 1,
+    paddingHorizontal: 16,
+  },
+  starTitle: {
+    alignItems: 'center',
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+    paddingBottom: 4,
+    paddingTop: 11,
+  },
+  textTitle: {
+    color: 'black',
+
```

Comment:
Please, pay attention to this part. Do not leave unnecessary blank lines. Try to make code more consistent.

Reply:
Okay! I thought eslint would help me to do this automatically. Because I deleted another fontfamily. It shows I could not have multiple font family;]